html-css/week1/Portfolio/css/styles.css

```
@@ -0,0 +1,686 @@
+    padding: 10px;
+    text-align: center;
+    border-radius: 200px;
+    height: 40px;
+    width: auto;
+    padding: 20px 30px;
+    display: flex;
+   
+}
+.nav-list{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 45px;
+    list-style: none;
+    text-align: center;
+    
+}
+.nav-list li a{
+    color: var(--drk-blu);
+    font-size: 1.2rem;
+    font-weight: 700;
+    text-decoration: none;
+    line-height:20px;
+    
+}
+/* h1, h2 */
+h1 {
+    color: var(--drk-blu);
+    margin-top: 20px;
+    margin-bottom: 10px;
+    font-size: 119px;
+    font-weight: 700;
+    line-height: 160px;
+  }
+
+  h2 {
+    color: var(--drk-blu);
+    display: block;
+    font-size: 60px;
+    font-weight: bold;
+   
+}
+
+  .hero p {
+    color: var(--drk-blu);
+    font-size: 2.5rem;
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  /* about section */
+
+ .section-heading {
+    position: relative;
+    font-size: 5.5rem;
+    font-weight: 700;
+    margin-top: 9rem;
+    margin-bottom: 13rem;
+    z-index: 9999;
+}
+.about-content{
+    display: flex;
+   justify-content: space-between;
+   gap: 40px;
+   width: 100%;
+}
+
+.profile {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: 500;
+   
+}
+
+
+.profile__picture {
+    height: 24rem;
+    margin-bottom: 4rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
+.profile__picture img {
+    height: 100%;
+    border-radius: 50%;
+    border: 5px solid #ff4d5a;
+    transition: all 0.2s;
+}
+
+.profile__blurb {
+    font-size: 1.8rem;
+    width: 500px;
+    line-height: 1.8
+}
+
+
+/* for skills */
+.skills {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 3rem;
+    margin-bottom: 3rem;
+}
+
+.skills__row{
+    display: flex;
+   flex-direction: column;
+   justify-content: center;
+   gap: 10px;
+}
+
+
+img {
+    overflow-clip-margin: content-box;
+    overflow: clip;
+}
+
+
+.skills__item {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 100px;
+    width: 160px;
+    border: double 3px transparent;
+    border-image-slice: 1;
+    border-width: 3px;
+    background-image: linear-gradient(#1a1a1a, #1a1a1a), radial-gradient(circle at top left, #51a2e9, #ff4d5a);
+    border-radius: 20px;
+    background-origin: border-box;
+    background-clip: content-box, border-box;
+    overflow: hidden;
+}
+
+
+
+.skills__item-name {
+    color: #fff;
+    font-size: 1.2rem;
+    font-weight: 600;
+    line-height: 1;
+    transition: all 0.2s;
+    letter-spacing: 1px;
+}
+.skills__item img{
+    height: 60px;
+    width: auto;
+    transition: all 0.2s
+}
+/* Project Sections */
+.projects {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+project {
```

```suggestion
.project {
```
You've forgotten the dot to indicate it's a class selector.